draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1079,15 +1079,32 @@ Decoy digests increase the size of the SD-JWT. The number of decoy digests (or w
 
 ## Unlinkability
 
-Colluding Issuer/Verifier or Verifier/Verifier pairs could link issuance/presentation
-or two presentation sessions to the same user on the basis of unique values encoded in the SD-JWT
-(Issuer signature, salts, digests, etc.).
-
-To prevent these types of linkability, various methods, including but not limited to the following ones can be used:
-
-- Use advanced cryptographic schemes, outside the scope of this specification.
-- Issue a batch of SD-JWTs to the Holder to enable the Holder to use a unique SD-JWT per Verifier. This only helps with Verifier/Verifier unlinkability.
-
+Unlinkability is a property that prevents an adversary from associating two or
+more sessions of the same user. Two types of unlinkability are often considered:
+
+- Verifier/Verifier unlinkability: Two colluding Verifiers should not be able to
+  tell that they have received presentations from the same user.
+- Issuer/Verifier unlinkability: An Issuer colluding with a Verifier should
+  not be able to tell that a user that has received a credential from the Issuer
+  has presented it to the Verifier.
+
+In both cases, unlinkability is limited to cases where the
+disclosed claims do not contain information that directly or indirectly
+identifies the user. For example, when a tax ID is contained in the disclosed
+claims, the Issuer and Verifier can easily link the user's sessions. However,
+when the user only discloses a birthdate to one Verifier and a postal code to
+another Verifier, the two Verifiers should not be able to tell that they were
+talking to the same user.
+
+Issuer/Verifier unlinkability cannot be achieved with SD-JWT, as the issued
+SD-JWT is directly forwarded to the Verifier.
```

Comment:
I don't understand this. Three party model in itself allows to realize issuer/verifier unlinkability. Are you trying to say that when the issuer directly forwards to the verifier it is not achieved? But in that case no need for sd-jwt.

```suggestion
```

Reply:
The three party model does not realize issuer/verifier unlinkability (per the definition above, it assumes colluding issuers and verifiers). This is one of the major differences between salted-hash based approaches and other approaches using more advanced cryptography.